pkg/pipeline/resource_pipeline_artifactory_integration.go

```diff
@@ -0,0 +1,127 @@
+package pipeline
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/jfrog/terraform-provider-shared/util"
+)
+
+func pipelineArtifactoryProjectIntegrationResource() *schema.Resource {
+
+	var artifactoryIntegrationSchema = util.MergeSchema(
+		projectIntegrationSchema,
+		map[string]*schema.Schema{
+			"apikey": {
+				Type:         schema.TypeString,
+				Required:     true,
+				Sensitive:    true,
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotEmpty),
+				Description:  "ApiKey for Artifactory access",
+			},
+			"user": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotEmpty),
+				Description:  "User for Artifactory access",
+			},
+			"url": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateDiagFunc: validation.ToDiagFunc(validation.IsURLWithScheme([]string{"github"})),
```

Comment:
mmm, what url exactly are you expecting here? You save with scheme `github` which would be `github://` but then in your description you have `URL to Artifactory instance` - none of this makes sense.